library/spark/spark-accumulo-library/src/main/java/uk/gov/gchq/gaffer/sparkaccumulo/operation/rfilereaderrdd/Utils.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2017 Crown Copyright
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.gov.gchq.gaffer.sparkaccumulo.operation.rfilereaderrdd;
+
+import org.apache.hadoop.conf.Configuration;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.DataInputStream;
+import java.io.DataOutputStream;
+import java.io.IOException;
+
+public final class Utils {
+
+    private Utils() {
+        // Empty
+    }
+
+    public static byte[] serialiseConfiguration(final Configuration configuration) throws IOException {
+        final ByteArrayOutputStream baos = new ByteArrayOutputStream();
```

Comment:
Please could you do this in a try-with-resources to ensure baos is always closed even if there is an exception. Similarly in deserialiseConfiguration.

Reply:
Done.